conans/client/cmd/uploader.py

```diff
@@ -1,11 +1,11 @@
+import os
 import time
-
-from conans import load
+from conans import tools
```

Comment:
```suggestion
from conans.util.files import load
```

Get rid of _globals_ polluting the code base.